packages/node/test/cron.test.ts

```diff
@@ -0,0 +1,59 @@
+import * as SentryCore from '@sentry/core';
+
+import { cron } from '../src';
+import type { NodeCron, NodeCronOptions } from '../src/cron/node-cron';
+
+describe('cron', () => {
+  let withMonitorSpy: jest.SpyInstance;
+
+  beforeEach(() => {
+    withMonitorSpy = jest.spyOn(SentryCore, 'withMonitor');
+  });
+
+  afterEach(() => {
+    jest.restoreAllMocks();
+  });
+
+  describe('node-cron', () => {
+    test('calls withMonitor', done => {
+      const nodeCron: NodeCron = {
+        schedule: (expression: string, callback: () => void, options?: NodeCronOptions): unknown => {
+          expect(expression).toBe('* * * Jan,Sep Sun');
+          expect(callback).toBeInstanceOf(Function);
+          expect(options?.name).toBe('my-cron-job');
+          return callback();
```

Comment:
we should add `expect.assertions` if we are asserting in callback like this.

Reply:
I thought that because it uses `done()` at the end, the test will fail if done isn't called?

Reply:
I guess technically that should make sure everything is called, but I'd rather have the `expect.assertions` for increased safety just in case.

Reply:
Will add it in!